packages/web/server/queries/complex/pools/providers/__tests__/sidecar.spec.ts

```diff
@@ -0,0 +1,226 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { rest } from "msw";
+
+import { server } from "~/tests/msw";
+
+import { calcAssetValue, getAsset } from "../../../assets";
+import { getPoolsFromSidecar } from "../sidecar";
+
+export const mockAsset = {
+  coinDenom: "mockCoinDenom",
+  coinName: "mockCoinName",
+  coinMinimalDenom: "mockCoinMinimalDenom",
+  coinDecimals: 0,
+  coinGeckoId: "mockCoinGeckoId",
+  coinImageUrl: "mockCoinImageUrl",
+  isVerified: true,
+};
+
+jest.mock("../../../assets", () => ({
+  getAsset: jest.fn(),
+  calcAssetValue: jest.fn(),
+}));
+
+describe("getPoolsFromSidecar", () => {
+  server.use(
+    rest.get("https://sqs.osmosis.zone/pools/all", (_req, res, ctx) => {
+      return res(ctx.json(mockSidecarResponse));
+    })
+  );
```

Comment:
This should be within the test or in a `beforeEach` hook because the msw handlers are reset after each test.